src/util/util.fh

```diff
@@ -14,9 +14,20 @@
 ! in f90 source files.                                                !
 !_____________________________________________________________________!
 
-#define INFILEHANDLE   2020
-#define OUTFILEHANDLE  2021
-#define DATAFILEHANDLE 2029
+#ifndef QUICK_UTIL_FH
+#define QUICK_UTIL_FH
+
+#define INFILEHANDLE     2020
+#define OUTFILEHANDLE    2021
+#define DMXFILEHANDLE    2022
+#define RSTFILEHANDLE    2023
+#define CPHFFILEHANDLE   2024
+#define BASISFILEHANDLE  2025
+#define ECPFILEHANDLE    2026
+#define BASISCFILEHANDLE 2027
+#define PDBFILEHANDLE    2028
+#define DATAFILEHANDLE   2029
+#define INTFILEHANDLE    2030
 
 ! For the following definitions to work, one must use quick_exception_module
 ! in a subroutine. For MPI/CUDAMPI versions, quick_mpi_module must be used. 
```

Comment:
In the line below, do you know why we don't do checks in the DEBUG version, i.e. when DEBUG is defined? That seems strange.

Reply:
I wasn't able to highlight the line as this is not a line in the code that was changed.